server/controllers/collectives.js

```diff
@@ -659,3 +699,22 @@ export const getTransactions = (req, res, next) => {
     })
     .catch(next);
 };
+
+/**
+ * Get array of all transactions of a collective given its slug
+ */
+export const getCollectiveTransactions = (req, res) => {
```

Comment:
Just a note for the future. Let's try to standardize async / await for our new code.

Reply:
agreed, did this way as if you look the other methods on this file, they use promise so was trying to keep at least the file standard but we know it's outdated code so follow that standard may not make sense anymore. Anyways, will create a new PR soon to fix a small thing and will add it. thanks a lot.